package.json

```diff
@@ -17,11 +17,11 @@
     "test:karma": "karma start",
     "test:coverage": "KARMA_ENV=coverage karma start --reporters coverage,coveralls",
     "watch:test": "karma start --no-single-run --browsers Firefox",
-    "test:http": "mocha tests/http/*.test.js",
+    "test:http": "NPM_CONFIG_PRODUCTION=true mocha tests/http/*.test.js",
```

Comment:
Hrm, it's not great that you need to set this. I filed https://github.com/mozilla/donate.mozilla.org/issues/1138